data/io.elementary.calculator.desktop.in

```diff
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Name=Calculator
 Comment=Solve simple equations
+Keywords=calculation;arithmetic;scientific;financial;
```

Comment:
Please start each keyword with a capital letter, per the [guide](https://elementary.io/docs/code/getting-started#the-desktop-file) :)